src/network/networkprotocol.h

```diff
@@ -987,12 +987,12 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
+	"The password is invalid.",
```

Comment:
“Your password is invalid.” is incorrect because it is not the (real) player password which is incorrect, but the password entered by the player.